Makefile

```diff
@@ -5,6 +5,23 @@ export ADDON_TESTS_PER_ADDON_WAIT_DURATION := 10m
 export GIT_TERMINAL_PROMPT := 1
 export ADDON_TESTS_SETUP_WAIT_DURATION := 30m
 export GOPRIVATE := github.com/mesosphere/kubeaddons
+export KUBECONFIG := kubeconfig
+
+KUTTL_VERSION=0.1.0
```

Comment:
FYI: kuttl version `0.2.0` was just released

Reply:
Just tested and assert is failing to match status fields. Will come back to this in a later iteration.

Reply:
`v0.3.0` is not now... and has some useful features for this PR